Clinical-Genomics/scout
# Omim disease missing

This PR fixes a bug.

If variant-gene objects were missing disease terms, these were not shown on the variants page tooltip.

**How to test**:
1. Try hovering over the gene names over a few genes with associated OMIM disorders. In all likelihood you will see no OMIM disorders on current master.
2. Do the same on current branch, and note that you now get the data fresh from hgnc_genes.

![Screenshot 2020-09-11 at 17 07 15](https://user-images.githubusercontent.com/758570/92942058-6c04be80-f451-11ea-8d12-9f77e907042c.png)

**Expected outcome**:
The functionality should be working
Take a screenshot and attach or copy/paste the output.

**Review:**
